### PR TITLE
nieuwe dataset afvalwijzer toegevoegd

### PR DESCRIPTION
**Pull request by ekesgin**

### CODE REVIEW

**Review by ekesgin — COMMENTED** [review]
spatialDescription aangepast en additionel properties is op true gezet.

File: datasets/afvalwijzer/dataset.json [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        "Duurzaamheid en milieu, wonen"
+    ],
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Gemeente Amsterdam, Afval en Grondstoffen",
+    "dateModified": "",
+    "spatialDescription": "Amsterdam",
+    "version": "1.0.0",
+    "title": "Afvalwijzer",
+    "language": "nl",
+    "dateCreated": "2022-01-01",
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:15]: This should work. I'll see if I can fix this on our side.

Comment (lbam) [reply datasets/afvalwijzer/dataset.json:15]: Nope, we should really use a date+time here for compatibility with DCAT.

```suggestion
    "dateCreated": "2022-01-01T00:00:00",
```

Comment (ekesgin) [reply datasets/afvalwijzer/dataset.json]: unfortunately still not accepting `"dateCreated": "2022-01-01T00:00:00+01:00",`

Comment (lbam) [reply datasets/afvalwijzer/dataset.json]: Works locally, let me try again:

```suggestion
    "dateCreated": "2022-01-01T00:00:00+01:00",
```

File: datasets/afvalwijzer/dataset.json [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+        "Duurzaamheid en milieu, wonen"
+    ],
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Gemeente Amsterdam, Afval en Grondstoffen",
+    "dateModified": "",
+    "spatialDescription": "Amsterdam",
+    "version": "1.0.0",
+    "title": "Afvalwijzer",
+    "language": "nl",
+    "dateCreated": "2022-01-01T00:00:00",
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:15]: This works locally, let me try to commit it:

```suggestion
    "dateCreated": "2022-01-01T00:00:00+01:00",

```

File: datasets/afvalwijzer/dataset.json [hunk 3]
```diff
@@ -0,0 +1,228 @@
+{
+    "id": "afvalwijzer",
+    "type": "dataset",
+    "authorizationGrantor": "n.v.t.",
+    "theme": [
+        "Duurzaamheid en milieu, wonen"
+    ],
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Gemeente Amsterdam, Afval en Grondstoffen",
+    "dateModified": "",
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:10]:
```suggestion
```

File: datasets/afvalwijzer/dataset.json [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+    "version": "1.0.0",
+    "title": "Afvalwijzer",
+    "language": "nl",
+    "dateCreated": "2022-01-01T00:00:00+01:00",
+
+    "license": "Creative Commons, Naamsvermelding",
+    "hasBeginning": "",
+    "hasEnd": "",
+    "accrualPeriodicity": "dagelijks",
+    "publisher": "OIS",
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:20]:
```suggestion
    "publisher": "datapunt@amsterdam.nl",
```

File: datasets/afvalwijzer/dataset.json [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+        "Duurzaamheid en milieu, wonen"
+    ],
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Gemeente Amsterdam, Afval en Grondstoffen",
+    "spatialDescription": "Amsterdam",
+    "version": "1.0.0",
+    "title": "Afvalwijzer",
+    "language": "nl",
+    "dateCreated": "2022-01-01T00:00:00+01:00",
+
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:15]:
```suggestion
```

File: datasets/afvalwijzer/dataset.json [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Gemeente Amsterdam, Afval en Grondstoffen",
+    "spatialDescription": "Amsterdam",
+    "version": "1.0.0",
+    "title": "Afvalwijzer",
+    "language": "nl",
+    "dateCreated": "2022-01-01T00:00:00+01:00",
+    "license": "Creative Commons, Naamsvermelding",
+    "hasBeginning": "",
+    "hasEnd": "",
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:17]:
```suggestion
```

File: datasets/afvalwijzer/dataset.json [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+    "keywords": [
+        "afval",
+        "restafval",
+        "glas",
+        "grofvuil",
+        "GFT",
+        "afvalwijzer"
+    ],
+    "crs": "EPSG:28992",
+    "objective": "De bewoner en ondernemer in staat te stellen om zich voor iedere fractie te informeren over de ophaaldata, -lokaties en wijzes van aanbieden ",
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:29]:
```suggestion
    "objective": "De bewoner en ondernemer in staat te stellen om zich voor iedere fractie te informeren over de ophaaldata, -locaties en wijzes van aanbieden",
```

File: datasets/afvalwijzer/dataset.json [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+                        "type": "string",
+                        "relation": "gebieden:buurten",
+                        "uri": "https://www.amsterdam.nl/stelselpedia/gebieden-index/catalogus/buurt",
+                        "description": "Unieke code (hierin zie je de Stadsdeel- en Wijkcode terug)"
+                    }
+                }
+            }
+        }
+    ]
+}
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:224]:
```suggestion
}

```

**Review by lbam — DISMISSED** [review]

File: datasets/afvalwijzer/dataset.json [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+      "description": "Overzicht van de ophaalinformatie per fractie en per locatie",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "schema",
+          "id"
+        ],
+        "mainGeometry": "geometrie",
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:50]:
```suggestion
        "mainGeometry": "afvalwijzerGeometrie",
```

File: datasets/afvalwijzer/dataset.json [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+          },
+          "afvalwijzerBuitenzettenTot": {
+            "type": "string",
+            "description": "Opmerking die aangeeft tot wannner de afval buiten mag gezet worden"
+          },
+          "afvalwijzerAfvalkalenderOpmerking": {
+            "type": "string",
+            "description": "Aanvullende opmerking over de manier van het aanbieden van het afval"
+          },
+          "afvalwijzerAfvalkalanderFrequentie": {
```
Comment (lbam) [inline datasets/afvalwijzer/dataset.json:127]:
```suggestion
          "afvalwijzerAfvalkalenderFrequentie": {
```

**Review by lbam — APPROVED** [review]
